Tribes/src/app/add-new-badge/add-new-badge.component.css

```diff
@@ -1,9 +1,13 @@
 .badge-add {
-    border-radius: 14px;
+	padding:3px;
+	height: 30px;
+	width: 30px;
+	border-radius: 15px;
+	line-height: 25px;
 	font-size: 30px;
 	position: relative;
-	top: -45px;
-	left: -10px;
+	top: -50px;
+	left: 0px;
 	color: #212529;
    	background-color: #ffc107;
```

Comment:
indentation fix pls

Reply:
Interesting, nothing comes up on my screen, indentation is consistent...